project/src/main/webapp/index.html

```diff
@@ -27,10 +28,103 @@
         width: 100%;
         height: 100%;
       }
+
+      /* Header/Logo Title */
+      .header {
+        height: 45px;
+        background: white;
+        color: white;
+        border: 1px solid grey
+      }
+
+      /* the toolbar with operations */
+      .operations {
+          height: 19px;
+          padding: 5px;
+          background: white;
+          border: 1px solid grey;
+      }
+
+      select {
+        -webkit-writing-mode: horizontal-tb !important;
+        text-rendering: auto;
+        color: -internal-light-dark-color(black, white);
+        letter-spacing: normal;
+        word-spacing: normal;
+        text-transform: none;
+        text-indent: 0px;
+        text-shadow: none;
+        display: inline-block;
+        text-align: start;
+        -webkit-appearance: menulist;
+        box-sizing: border-box;
+        align-items: center;
+        white-space: pre;
+        -webkit-rtl-ordering: logical;
+        background-color: -internal-light-dark-color(white, black);
+        cursor: default;
+        margin: 0em;
+        font: 400 13.3333px Arial;
+        border-radius: 0px;
+        border-width: 1px;
+        border-style: solid;
+        border-color: rgb(169, 169, 169);
+        border-image: initial;
+      }
+
     </style>
   </head>
 
   <body onload="init()">
+    <div class="header"></div>
+    <div class="operations">
+      Theme: 
+      <select id="select">
+        <option selected>ayu-dark</option>
+      </select>
+    </div>
     <div id="firepad-container"></div>
+
+    <script>
```

Comment:
Similar to my other comment, is there a particular reason that these scripts are not in another file?